experimental/rsocket-src/RSocketRequester.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include "rsocket/RSocketRequester.h"
+
+using namespace folly;
+
+namespace rsocket {
+
+std::shared_ptr<RSocketRequester> RSocketRequester::create(
+    std::unique_ptr<StandardReactiveSocket> srs,
+    EventBase& eventBase) {
+  auto customDeleter = [&eventBase](RSocketRequester* pRequester) {
+    eventBase.runImmediatelyOrRunInEventBaseThreadAndWait([&pRequester] {
+      LOG(INFO) << "RSocketRequester => destroy on EventBase";
+      delete pRequester;
+    });
+  };
+
+  auto rsr = new RSocketRequester(std::move(srs), eventBase);
+  std::shared_ptr<RSocketRequester> sR(rsr, customDeleter);
+  return sR;
+}
+
+RSocketRequester::RSocketRequester(
+    std::unique_ptr<StandardReactiveSocket> srs,
+    EventBase& eventBase)
+    : standardReactiveSocket_(std::move(srs)), eventBase_(eventBase) {}
+
+RSocketRequester::~RSocketRequester() {
+  LOG(INFO) << "RSocketRequester => destroy";
+}
+
+std::shared_ptr<Subscriber<Payload>> RSocketRequester::requestChannel(
+    std::shared_ptr<Subscriber<Payload>> responseSink) {
+  return standardReactiveSocket_->requestChannel(std::move(responseSink));
```

Comment:
Yes, I forgot to leave the TODO. I didn't have time to figure this one out with its current signature.